### PR TITLE
Fixes #211: Link Preview View reloading resolved

### LINKED ISSUE #211: Link preview reloads after its brought back in view
**Actual Behaviour**

Link preview reloads.

**Expected Behaviour**

Link preview should be cached atleast till app is destroyed.

**Steps to reproduce it**

send a link from your side, a preview would be shown.
scroll a bit up such that link message is out of view, then go to link message it reloads.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by betterclever**
Fixes issue #211 : It solves the issue of reloading of Link Preview view when it is brought back in view.

Changes: The Message Model was updated to include webpreview info so it is not loaded again and again using jsoup but only once.

Screenshots for the change: 

ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -928,8 +928,8 @@ private void updateDatabase(final long id, final String message, final String da
         realm.executeTransactionAsync(new Realm.Transaction() {
             @Override
             public void execute(Realm bgRealm) {
-                ChatMessage chatMessage = bgRealm.createObject(ChatMessage.class);
-                chatMessage.setId(id);
+                ChatMessage chatMessage = bgRealm.createObject(ChatMessage.class,id);
+                //chatMessage.setId(id);
```
Comment (daminisatya) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Remove this

Comment (betterclever) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Okay

**Comment by betterclever** [comment]
@daminisatya Circle CI is failing for PR. I'll send a fix soon.

**Comment by daminisatya** [comment]
sure.

**Pull request merged**